frontend/app/controllers/CachedAssets.scala

```diff
@@ -4,7 +4,7 @@ import play.Logger
 import play.api.mvc.{Action, Controller}
 import play.api.libs.concurrent.Execution.Implicits.defaultContext
 
-object CachedAssets extends Controller {
+class CachedAssets() extends Controller {
```

Comment:
just a heads up when I switched to play 2.6 this started 404ing all the assets in dev, so I had to change to DI for the Assets controller into this class.

Reply:
Good to know, thanks